src/test/java/teammates/test/cases/util/TimeHelperTest.java

```diff
@@ -105,4 +105,10 @@ public void testConvertLocalDateToUtc() {
         assertNull(TimeHelper.convertLocalDateToUtc(null, offsetHours));
     }
 
+    @Test
+    public void testformatDateForInstructorPages() {
```

Comment:
I feel that we should add this test case before `testFormatDateTimeForDisplay` as they are basically testing on the same thing (one date format, one date&time format).

Reply:
Added the additional test.